config/initializers/logstasher.rb

```diff
@@ -101,6 +99,8 @@ def self.replace_known_types!( hash )
       end
       hash.delete( :user )
     end
+    # set default value to logged_in if no value was set
+    hash[:logged_in] = false if hash[:logged_in].nil?
```

Comment:
My recommendation would be to put this line inside the conditional above `if hash[:request].is_a?( ActionDispatch::Request )`, under `hash.merge!( Logstasher.payload_from_request( hash[:request] ) )`. That would most closely fix the existing behavior. Other logging events (delayed jobs, exceptions, etc) call `replace_known_types!` and those logs did not previously include a `logged_in` attribute. With this PR as submitted, they would begin including a `logged_in` attribute. The goal is to modify behavior that was coming from `payload_from_request`, so best to limit the fix to where that is used.

With that change, this would resolve the issue and be OK to merge.

Reply:
Indeed, it will make less behavior change.